src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts

```diff
@@ -34,7 +34,7 @@ export interface NameRequestIF {
   details?: NameRequestDetailsIF
   expiry?: string
   filingId?: number
-  legalName: string
+  legalName?: string
```

Comment:
This is needed to prevent a type error in `filing-template-mixins.ts` at `FilingTemplateMixin::parseCorrectionFiling`. When it tries to store the name request with `this.setNameRequest`, the `filing.correction.nameRequest` has `legalName` as an optional value.

Reply:
Leo, you can resolve this conversation (unless you want it visible still).